packages/backend/src/libp2p/connectionsManager.ts

```diff
@@ -300,15 +307,19 @@ export class ConnectionsManager extends EventEmitter {
         detached: true
       }
     })
+
+    await this.initTorMobile()
+  }
+
+  public initTorDesktop = async () => {
+    if (this.torBinaryPath) {
+      await this.tor.init()
+    }
   }
 
-  public initTor = async () => {
+  public initTorMobile = async () => {
     if (this.torControlPort) {
       this.tor.initTorControl()
-    } else if (this.torBinaryPath) {
-      await this.tor.init()
-    } else {
-      throw new Error('You must provide either tor control port or tor binary path')
```

Comment:
aren't we loosing this throw by separating the method into `initTorDesktop` and `initTorMobile`?